eng/Subsets.props

```diff
@@ -54,7 +54,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <DefaultCoreClrSubsets>clr.runtime+clr.jit+clr.alljits+linuxdac+clr.corelib+clr.nativecorelib+clr.tools+clr.packages</DefaultCoreClrSubsets>
+    <DefaultCoreClrSubsets>clr.runtime+clr.jit+clr.alljits+linuxdac+clr.corelib+clr.tools+clr.nativecorelib+clr.packages</DefaultCoreClrSubsets>
```

Comment:
Does the order matter here?

Reply:
Yes it does, with the change "clr.tools" is now a pre-requisite of "clr.nativecorelib". I also had to remove the "BuildInParallel" clause for the "tools" subset because the way the project collection and execution works today it ended up decoupling the "parallelizable" tools with the subsequent "nativecorelib" and naturally failing.